lib/system/web/js/resize_observer.js

```diff
@@ -54,12 +54,12 @@ class Pool {
 
 let resizeObserverPool = new Pool((...args) => new ResizeObserver(...args))
 
-export function resize_observe(target, f) {
+function resize_observe(target, f) {
     let id = resizeObserverPool.reserve(resize_observer_update(f))
     resizeObserverPool[id].observe(target)
 }
 
-export function resize_unobserve(id) {
+function resize_unobserve(id) {
```

Comment:
Why those changes?

Reply:
Once I changed test in core from #[test] to #[wasm_bindgen_test] and it turns out that wasm-pack test does not work with "export" word (even with --node option...). But I don't know why, I don't have a deep knowledge of Js.

Reply:
The exact error is ```Running /home/adam-praca/dev/basegl/target/wasm32-unknown-unknown/debug/deps/basegl-c02b39c2bb2e07d5.wasm
/home/adam-praca/dev/basegl/target/wasm32-unknown-unknown/wbg-tmp/snippets/basegl-system-web-056d2c0780aef067/js/resize_observer.js:57
export function resize_observe(target, f) {
^^^^^^

SyntaxError: Unexpected token 'export'
```
But without exports the example 01-scene does not work. Now I'm putting version which works on both browser and wasm-pack test

Reply:
strange :/